warehouse/integrations/azure-synapse/azure-synapse.go

```diff
@@ -625,7 +626,7 @@ func (as *AzureSynapse) AddColumns(tableName string, columnsInfo []warehouseutil
 	))
 
 	for _, columnInfo := range columnsInfo {
-		queryBuilder.WriteString(fmt.Sprintf(` %s %s,`, columnInfo.Name, rudderDataTypesMapToMssql[columnInfo.Type]))
+		queryBuilder.WriteString(fmt.Sprintf(` %q %s,`, columnInfo.Name, rudderDataTypesMapToMssql[columnInfo.Type]))
```

Comment:
This got missed when we added support for skipping escaping keywords. https://github.com/rudderlabs/rudder-server/pull/2115